Apps/WebClient/src/ClientApp/src/components/private/sharing/DelegateInvitationDialog.vue

```diff
@@ -0,0 +1,186 @@
+<script setup lang="ts">
+import { computed, ref } from "vue";
+
+import HgButtonComponent from "@/components/common/HgButtonComponent.vue";
+import HgIconButtonComponent from "@/components/common/HgIconButtonComponent.vue";
+import InvitationContactStepComponent from "@/components/private/sharing/InvitationContactStepComponent.vue";
+import InvitationExpiryStepComponent from "@/components/private/sharing/InvitationExpiryStepComponent.vue";
+import InvitationRecordTypesStepComponent from "@/components/private/sharing/InvitationRecordTypesStepComponent.vue";
+import InvitationReviewStepComponent from "@/components/private/sharing/InvitationReviewStepComponent.vue";
+import InvitationSharingCodeStepComponent from "@/components/private/sharing/InvitationSharingCodeStepComponent.vue";
+import {
+    DelegateInvitationWizardStep,
+    useDelegateStore,
+} from "@/stores/delegate";
+
+const delegateStore = useDelegateStore();
+
+const invitationContactStepComponent =
+    ref<InstanceType<typeof InvitationContactStepComponent>>();
+const invitationRecordTypesStepComponent =
+    ref<InstanceType<typeof InvitationRecordTypesStepComponent>>();
+const invitationExpiryStepComponent =
+    ref<InstanceType<typeof InvitationExpiryStepComponent>>();
```

Comment:
Suggest renaming variable and component to `delegationExpiryStepComponent` / `DelegationExpiryStepComponent`.  The other step components could stay as-is or follow the same convention for consistency.